Packs/Slack/Scripts/SlackBlockBuilder/README.md

```diff
@@ -78,3 +95,34 @@ Thread ID is: 1660645689.649679
 Message sent to Slack successfully.
 Thread ID is: 1660645689.649679
 ```
+
+## Troubleshooting
+---
+**Issue**: The survey message is not sent to Slack at all.
+
+**Troubleshooting**: Test the `send-notification` command on its own and get it working first. That will provide a more specific error message if there is an issue. Note that the Slack API bot you set up for the Slack v3 integration needs to be added to any channels you want to send SlackBlockBuilder surveys to.
```

Comment:
```suggestion
**Troubleshooting**: Test the `send-notification` command on its own to verify it is working correctly. This will provide a more specific error message if there is an issue. Note that the Slack API bot you set up for the Slack v3 integration needs to be added to any channels you want to send SlackBlockBuilder surveys to.
```